Unix/tools/auth_tests_cleanup.sh

```diff
@@ -2,12 +2,12 @@
 # 
 # output directory and its softlink target are the two arguments
 
-ntlm_file=$1/tmp/ntlm
+#ntlm_file=$1/etc/ntlm
 
-if [ -f $ntlm_file ]; then
-    echo "Removing NTLM file: $ntlm_file"
-    rm -f $ntlm_file
-fi
+#if [ -f $ntlm_file ]; then
+#    echo "Removing NTLM file: $ntlm_file"
+#    sudo rm -f $ntlm_file
+#fi
```

Comment:
So why is this block commented out? Should it just be removed entirely? Sometimes it's useful to check in a commented-out block, but often it's just because we forgot to do the "right thing" (fix it or get rid of it).